WordPress/src/main/res/layout/blogging_prompts_list_fragment.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/coordinator"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <TextView
+        android:id="@+id/temp_text"
+        android:layout_width="wrap_content"
+        android:layout_gravity="center"
+        android:layout_height="wrap_content"
+        android:text="PROMPTS LIST PAGE" />
```

Comment:
Since we're using `android:text`, should we extract this to `strings.xml` file? In case this is just for the XML preview, maybe we could use `tools:text`? What do you think?

Reply:
This is temporary. It's only to check if the arguments are being passed to the screen. (Check if it's the answered fragment or unanswered fragment). This will be replaced by a recyclerview in the next PR.

Reply:
That said, I've changed the code to `tools:text`. You're right, the label "PROMPTS LIST PAGE" was only used as a placeholder for XML preview.